### PR TITLE
Reserves additions

### PR DESCRIPTION
**Pull request by michalisFr**
Changes in this PR:
- Added `https:` in front of the custom scripts (couldn't make it work otherwise)
- Set default address to mine and RPC to Polkadot in `index.html` (to test)
- Changed the display of all results to floats in the unit of the chain
- Fixed the decimals calculations in `toUnit`: leading zeros in the decimals weren't accounted for. Also, added a check for a negative result (from difference)
- Added `registrar` and `crowdloan` pallets
- Added reserves from sub-identities
- Replaced the `electionsPhragmen` pallet with `phragmenElection`

### CODE REVIEW

File: reserved.js [hunk 1]
```diff
@@ -327,8 +404,11 @@ async function calculateReserved() {
 
 	// Calculate the reserved balance pallet to pallet
 	reserved = reserved.add(await getDemocracyReserved());
-	reserved = reserved.add(await getElectionsReserved());
+	//reserved = reserved.add(await getElectionsReserved());
```
Comment (shawntabrizi) [inline reserved.js]: Why is this commented out?

We can't have both, or?

File: reserved.js [hunk 2]
```diff
@@ -67,10 +78,11 @@ async function getDemocracyReserved() {
...
 	}
 
-	output.innerText += `Democracy: Deposit = ${deposit}, Preimage = ${preimageDeposit}\n`;
+	output.innerText += `Democracy: Deposit = ${toUnit(deposit)}, Preimage = ${toUnit(preimageDeposit)}\n`;
 	return deposit.add(preimageDeposit);
 }
 
+/*
 async function getElectionsReserved() {
 	if (!substrate.query.electionsPhragmen) {
```
Comment (shawntabrizi) [inline reserved.js]: why was this commented out?

File: reserved.js (hunk 1 above)
Comment (michalisFr) [reply reserved.js]: I don't think that pallet exists anymore. I think it's been replaced by `phragmenElection`. But I didn't want to delete anything

File: reserved.js (hunk 2 above)
Comment (michalisFr) [reply reserved.js]: Same reason as above

File: reserved.js (hunk 1 above)
Comment (shawntabrizi) [reply reserved.js]: does the program panic though when this is in there?

We should keep it in, and simply return 0 deposit if the pallet doesn't exist

Comment (michalisFr) [reply reserved.js]: No it does not. It simply logs in the console `no elections pallet`.

Sure, I'll add it back

Comment (michalisFr) [reply reserved.js]: Done

**Review by shawntabrizi — APPROVED** [review]

**Pull request merged**